lib/common.sh

```diff
@@ -251,16 +249,16 @@ export IRONIC_NAMESPACE=${IRONIC_NAMESPACE:-"baremetal-operator-system"}
 export NAMEPREFIX=${NAMEPREFIX:-"baremetal-operator"}
 
 # CAPM3 and IPAM controller images
-if [ "${CAPM3RELEASEBRANCH}" == "release-0.5" ] || [ "${CAPM3_VERSION}" == "v1alpha5" ]; then
+if [[ "${CAPM3RELEASEBRANCH}" == "release-0.5" ]]; then
```

Comment:
As v1alpha5 and release-0.5 point to the same thing while release-0.5 still needed by upgrade just thinking what in some places we could clean it as well  ?

Reply:
I'd vote to merge this one first, and address that issue in another PR, as it's likely to require changes in other repos as well. How about it?